src/relay/ir/pretty_printer.cc

```diff
@@ -46,8 +46,7 @@ class PrettyPrinter :
     Doc PrintFinal(const NodeRef& node) {
       // TODO(@jmp): If these lines are combined it segfaults??
```

Comment:
fix the todo.
doc_stack_.back() is a reference which get invalidate as Print might push_back into doc_stack_
using a linked list might also fix the problem